### ISSUE TITLE
[Disambiguation] - Improve readability of statistics reported

### ISSUE THREAD

**Issue by phron3sis** [Bot-detector/bot-detector#25]
For example:
"Reports Made" could be renamed "Names Uploaded" and Names Uploaded could simply be clarified to be current session stats.
https://i.imgur.com/kdl0xaz.png

If you look at the img I linked you can see reports made is showing 4663. That seems like I "reported" that many players. Perhaps add details of how many players i reported in-game, reports via bot detector plugin, and name uploads/submissions.

Basically asking for more info and clarification of info already displayed.

**Comment by extreme4all** [comment]
Thanks for the feedback, i added it to the project board

**Comment by Cyborger1** [comment]
Most of the labels have been changed in the dev version
